src/Manager/Environment/Name.tsx

```diff
@@ -6,15 +6,39 @@
 
 import React, { FC } from 'react';
 import Row from 'react-bootstrap/Row';
+import { execSync } from 'child_process';
+import path from 'path';
 
+import { checkExecArchitecture } from '../../helpers';
 import { Environment } from '../../state';
 
 import './style.scss';
 
-const Name: FC<{ environment: Environment }> = ({ environment }) => (
-    <Row noGutters className="toolchain-item-info h4 mb-0 pt-3">
-        nRF Connect SDK {environment.version}
-    </Row>
-);
+const isAppleSilicon =
+    process.platform === 'darwin' && execSync('uname -m').includes('arm64');
+
+const Name: FC<{ environment: Environment }> = ({ environment }) => {
+    let arch = '';
+    if (isAppleSilicon && environment.isInstalled) {
+        try {
+            const toolchain = execSync(
+                `file $(find  ${path.join(
+                    environment.toolchainDir,
+                    'Cellar',
+                    'ninja'
+                )} -type f -name ninja)`
+            );
+
+            arch = checkExecArchitecture(toolchain.toString());
+        } catch {
+            arch = 'unknown';
```

Comment:
just displaying "unknown" might be unclear to the users.
```suggestion
            arch = 'unknown architecture';
```

Reply:
other option would be not displaying this hint at all as path may change for example

also I suggest adding this to regression checklist as it may be broken in future SDK releases (will add myself)